cmake/AwsPrebuildDependency.cmake

```diff
@@ -39,11 +38,22 @@ function(aws_prebuild_dependency)
         list(APPEND cmakeCommand ${AWS_PREBUILD_CMAKE_ARGUMENTS})
     endif()
 
-    list(APPEND cmakeCommand WORKING_DIRECTORY ${depBinaryDir})
-    list(APPEND cmakeCommand RESULT_VARIABLE result)
+    # Set cmake prefix path via envrionment variable incase it's already a list.
+    # Some version of cmake will treat the values in the list as separate args.
+    if(WIN32)
+        list(JOIN CMAKE_PREFIX_PATH ";" PREFIX_PATH_ENV_VAR)
+    else()
+        list(JOIN CMAKE_PREFIX_PATH ":" PREFIX_PATH_ENV_VAR)
```

Comment:
[list(JOIN)](https://cmake.org/cmake/help/latest/command/list.html) is cmake 3.12+